### ISSUE TITLE
Make link available that always points at the latest AppImage

### ISSUE THREAD

**Issue by r-cheologist** [mooltipass/moolticute#1166]
For scripted installs I would appreciate a link that always points at the latest `*.AppImage`.

`https://github.com/mooltipass/moolticute/releases/latest` gets me half way there, but the `*.AppImage` name also contains the version number, which is not resolved by that link...

**Comment by limpkin** [comment]
hello! why not parse the updater.json?

**Comment by r-cheologist** [comment]
Good idea, but the current one is at https://github.com/mooltipass/moolticute/releases/download/v1.01.0/updater.json,.... Where to find a generic or `latest` link airways to the latest one?

**Comment by limpkin** [comment]
https://github.com/mooltipass/moolticute/releases/latest/download/updater.json

**Comment by r-cheologist** [comment]
VERY COOL. Thank you.
